apps/pair.py

```diff
@@ -278,10 +302,14 @@ async def pair(sc, mitm, bond, io, prompt, request, print_keys, keystore_file, d
 
             if not request:
                 try:
-                    await connection.pair()
+                    if mode == 'le':
+                        await connection.pair()
+                    else:
+                        await connection.authenticate()
+                    return
```

Comment:
As I mentioned in another comment, it would be great to hide the difference to the user

Reply:
I agree. This started as two separate methods, mostly because initially only BLE pairing was implemented, then basic Classic authentication was added. I propose that we start with the two APIs as they are for now, and in a future PR unify them in a single API.

Reply:
Sounds good!